client/components/payment-methods-checkboxes/payment-method-checkbox.js

```diff
@@ -3,16 +3,51 @@
  * External dependencies
  */
 import React from 'react';
-import { CheckboxControl, Icon } from '@wordpress/components';
+import { CheckboxControl, Icon, VisuallyHidden } from '@wordpress/components';
 import { useCallback, useMemo } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import PaymentMethodIcon from '../../settings/payment-method-icon';
 import Pill from '../pill';
+import Tooltip from '../tooltip';
 import './payment-method-checkbox.scss';
 
+const infoMap = {
```

Comment:
Yeah, I think at some point it might be useful to extract things to separate consts (at least the payment method enums).
But I think it's good to colocate what is needed rather than placing everything in a "constants" file - so for this reason I'd rather keep this map here for now, since it's not needed anywhere else :)

Reply:
I misunderstood the comment, sorry about that! Fixed in 4d5c1cdc